terraform/modules/department/50-aws-iam-policies.tf

```diff
@@ -758,3 +748,28 @@ data "aws_iam_policy_document" "redshift_department_read_access" {
     resources = ["*"]
   }
 }
+
+
+// Glue job runner pass role to glue for notebook use
+data "aws_iam_policy_document" "glue_runner_pass_role_to_glue_for_notebook_use" {
+  statement {
+    effect = "Allow"
+    actions = [
+      "iam:PassRole"
+    ]
+    resources = [
+      aws_iam_role.glue_agent.arn
+    ]
+    condition {
+      test     = "StringEquals"
+      variable = "iam:PassedToService"
+      values   = ["glue.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_iam_policy" "glue_runner_pass_role_to_glue_for_notebook_use" {
+  tags   = var.tags
```

Comment:
we can add "count = var.environment == "prod" ? 0 : 1" here too to avoid create this policy in prod.

Reply:
yeah, I'll update the variable description - that's how I landed up with Prod in one of my earlier commits and you're right, I should have updated the description too. 

Creating the policies in prod I'm inclined to leave them. No one should be manually attaching policies in production anyway and it's one less thing to change if we wanted to do a notebook set up for production down the line (but that would obviously need a much more thought through implementation).

Reply:
Cheers